### PR TITLE
Fix getting default

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/bootloader/grub2base.rb [hunk 1]
```diff
@@ -120,6 +120,9 @@ def propose
 
       propose_serial
 
+      # we proposing new config and we do not know sections yet, so propose empty
```
Comment (afaerber) [inline src/lib/bootloader/grub2base.rb]: "we are proposing a new config"

Comment (afaerber) [inline src/lib/bootloader/grub2base.rb]: "propose empty" reads weird, but I don't have a better suggestion

Comment (jreidinger) [inline src/lib/bootloader/grub2base.rb]: "maybe propose no default"?

File: src/lib/bootloader/grub2base.rb [hunk 2]
```diff
@@ -120,6 +120,9 @@ def propose
...
+      @sections.default = ""
```
Comment (mvidner) [inline src/lib/bootloader/grub2base.rb]: Isn't this a bug in the constructor of Sections? IMHO this code should go to https://github.com/yast/yast-bootloader/blob/f519e4818cecaea9f25c52e63a41b7b0692be0b5/src/lib/bootloader/sections.rb#L20

Comment (jreidinger) [inline src/lib/bootloader/grub2base.rb]: well, sections by default after construction do lazy loading of settings. It is question of default if we should read it in constructor? but then we have to have some read method, which now do not exist.

File: src/lib/bootloader/grub2base.rb (hunk 1 above)
Comment (afaerber) [inline src/lib/bootloader/grub2base.rb]: Sure. (any noun will do)

File: src/lib/bootloader/grub2base.rb (hunk 2 above)
Comment (mvidner) [inline src/lib/bootloader/grub2base.rb]: I still can't quite put my finger on it. Here we're changing `sections.default ` from "unknown" to "empty" so we should do the same to `sections.all`. Either here or in the constructor.

Comment (mvidner) [inline src/lib/bootloader/grub2base.rb]: To put it in yet another way: I suspect that we do not have a consistent state, either before this assignment, or after it.

**Comment by jreidinger** [comment]
@mvidner yes, you understand correctly. So lets merge it

**Pull request merged**